src/data/posts/fountain-beta.md

```diff
@@ -0,0 +1,88 @@
+Today, we're proud to announce that we open the beta version of the Yeoman Fountain generators.
+
+Basically, it's a new, state of the art, Yeoman generator. It's also, to our knowledge, the most ambitious one as it's not a generator for one technology stack but perhaps one to rules them all. More seriously, one for any webapp with the most popular frameworks and tools.
+
+Using this generator will ask you for several, very structurant, questions: the Web framework you want to use, the module management, the JS and CSS preprocessors you want.
```

Comment:
Je mettrais plutôt : Using this generator, you'll be asked several, very structurant, questions...